network/richlist_symbol.py

```diff
@@ -54,38 +56,37 @@ def _prepare_nodes(self):
         builder.build()
         self.public_key_to_descriptor_map = builder.peers_map
 
-        log.info('found {} mappings'.format(len(self.public_key_to_descriptor_map)))
+    def _download_finalization_information(self):
+        self.finalization_epoch = self.api_client.get_finalization_info().epoch
+        self.voters_map = self.api_client.get_voters(self.finalization_epoch)
```

Comment:
not sure if this should be current or previous (-1) epoch

Reply:
_neither_, current might be missing votes (iirc), previous might be misleading

(but leaning towards previous)